styles/layout-style.module.scss

```diff
@@ -7,4 +7,5 @@
 
 .main {
   margin: 20px;
+  font-family: monospace;
```

Comment:
NIT: use the same font throughout, unless required so.

Reply:
.main class i a wrapper for whole web-app; which means that its a single source of font-family.

Reply:
Then better let it be in global styles file.

Reply:
.main itself is a wrapper component. The intension is to have the parent styles in the parent component. 
Also, request you to be be polite in ur comments and we are trying to help each other and not teach each other. Holding a simple PR for NIT comments is not suggested. Hope u get the polite message.

Reply:
Sorry if I sounded rude. I am not doing this to be sounded rude either. Since this is a learning process, I am too learning along with you. No offense intended.

Reply:
@ankushdharkar request for ur review

Reply:
I lack context here. @tejasberi Could you explain to us please why we are using `monospace` font separately from what's coming in from the site?

Reply:
i have tried to play with a new font here. seem to be looking good. suggest u to gave a look. if that does not suit the site style guide, i will revert it.

Reply:
Got it. Thanks